docs/examples/DataSet/Offline Plotting Tutorial.ipynb

```diff
@@ -11019,26 +11026,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "By default Matplotlib renders each individual data point as a separate square in 2D plots when storing in a vector format (pdf,svg). This is not a problem for small data sets, but the time needed to generate a pdf increases rapidly with the number of data points. Therefore, `plot_dataset` will automatically rasterize the data (lines, ticks and labels are still stored as text) if more than 5000 data points are plotted. The particular value of the rasterization threshold can be set in the `qcodesrc.json` config file.\n",
+    "By default `matplotlib` renders each individual data point as a separate square in 2D plots when storing in a vector format (pdf,svg). This is not a problem for small data sets, but the time needed to generate a pdf increases rapidly with the number of data points. Therefore, the `plot_dataset` will automatically rasterize the data (lines, ticks and labels are still stored as text) if more than 5000 data points are plotted. The particular value of the rasterization threshold can be set in the `qcodesrc.json` config file.\n",
```

Comment:
```suggestion
    "By default `matplotlib` renders each individual data point as a separate square in 2D plots when storing in a vector format (pdf, svg). This is not a problem for small data sets, but the time needed to generate a pdf increases rapidly with the number of data points. Therefore, the `plot_dataset` will automatically raster the data (lines, ticks and labels are still stored as text) if more than 5000 data points are plotted. The particular value of the rasterization threshold can be set in the `qcodesrc.json` config file.\n",
```

Reply:
@astafan8 I am not seeing a difference here?

Reply:
it's a space between `(pdf, svg)`